proto/store/data_source.proto

```diff
@@ -14,17 +14,15 @@ message DataSourceOptions {
   // sid and service_name are used for Oracle.
   string sid = 3;
   string service_name = 4;
-}
-
-message SSHConfig {
-  // The hostname of the SSH server agent.
-  string host = 1;
-  // The port of the SSH server agent. It's 22 typically.
-  string port = 2;
-  // The user to login the server.
-  string user = 3;
-  // The password to login the server. If it's empty string, no password is required.
-  string password = 4;
-  // The private key to login the server. If it's empty string, we will use the system default private key from os.Getenv("SSH_AUTH_SOCK").
-  string private_key = 5;
-}
+  // SSH related
+   // The hostname of the SSH server agent.
+   string ssh_host = 5;
+   // The port of the SSH server agent. It's 22 typically.
+   string ssh_port = 6;
+   // The user to login the server.
+   string ssh_user = 7;
+   // The password to login the server. If it's empty string, no password is required.
+   string ssh_obfuscated_password = 8;
```

Comment:
Can we name it ssh_password? 🤔️

Reply:
Since we store obfuscated format, let's name it this way. We should have done it fo data source password column.